cmd/lima

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 set -eu
 : "${LIMA_INSTANCE:=default}"
+: "${LIMA_SHELL:=}"
```

Comment:
Wondering rather we should have a generic way to support `limactl shell` options in the `lima` script. Perhaps like `LIMA_SHELL_ARGS=--workdir=/ --shell=/bin/zsh`. But on the second thought maybe users should just set `alias lima=limactl shell ....`.


_Originally posted by @AkihiroSuda in https://github.com/lima-vm/lima/pull/779#discussion_r843467813_

Reply:
> should have a generic way to support `limactl shell` options in the `lima` script

Practically speaking, we only have 2 options: `--shell` and `--workdir`, and I don't really expect somebody to hard-code the workdir for each `lima` call. But even then, I would rather add another `LIMA_WORKDIR` variable instead, so the options can be manipulated independent of each other.

Reply:
> we only have 2 options

Eventually we will have more

Reply:
Yeah, but I still like having individual variables. This is how I would do it:

```shell
set - "$LIMA_INSTANCE" "$@"
if [ -n "${LIMA_SHELL}" ]; then
  set - --shell "$LIMA_SHELL" "$@"
fi
if [ -n "${LIMA_WORKDIR}" ]; then
  set - --workdir "$LIMA_WORKDIR" "$@"
fi
exec "$LIMACTL" shell "$@"
```

This is trivially extensible to additional settings.

Reply:
I've updated the PR to include support `LIMA_WORKDIR` as well (and rebased on top of `master`, so the `--shell` option is actually functional.